rackspace/rackconnect/v3/cloudnetworks/requests.go

```diff
@@ -0,0 +1,27 @@
+package cloudnetworks
+
+import (
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// List returns all cloud networks that are associated with RackConnect. The ID
+// returned for each network is the same as the ID returned by the networks package.
+func List(c *gophercloud.ServiceClient) pagination.Pager {
+	url := listURL(c)
+	createPage := func(r pagination.PageResult) pagination.Page {
+		return CloudNetworkPage{pagination.SinglePageBase(r)}
+	}
+	return pagination.NewPager(c, url, createPage)
+}
+
+// Get retrieves a specific cloud network (that is associated with RackConnect)
+// based on its unique ID.
+func Get(c *gophercloud.ServiceClient, id string) GetResult {
+	var res GetResult
+	_, res.Err = c.Request("GET", getURL(c, id), gophercloud.RequestOpts{
+		JSONResponse: &res.Body,
+		OkCodes:      []int{200},
```

Comment:
:+1: I'm on board with this.

Something else I've thought about is providing the ability to map other status codes to concrete errors, so a `Get` call could return `ErrContainerNotFound` instead of the more generic "unexpected status code" error. The benefit to doing this would be that it would be easier for client code to catch common error conditions and recover appropriately -- as another concrete example, it's currently awkward to report when authentication has failed.

Let's make a pair of issues about this :point_right: